### ISSUE TITLE
PrismicRichText Not working with filed !

### ISSUE THREAD

**Issue by HaribolGM** [a-trost/creative-portfolio#1]
If you trying to run this code down below it's working but in the code as PrismicRichText attributre is not working with filed 

const Biography = ({ slice }: BiographyProps): JSX.Element => {
  return (
    <Bounded
      data-slice-type={slice.slice_type}
      data-slice-variation={slice.variation}
    >
      <div className="grid gap-x-8 gap-y-6 md:grid-cols-[2fr,1fr]">
        <Heading size="xl" className="col-start-1">
          {slice.primary.heading}
        </Heading>

        <div className="prose prose-xl prose-slate prose-invert col-start-1">
          {slice.primary.description} 
        </div>
       

       
      </div>
    </Bounded>
  );
};

export default Biography;

&& 
when i trying to run this code 

I am getting error  

code: 

const Biography = ({ slice }: BiographyProps): JSX.Element => {
  return (
    <Bounded
      data-slice-type={slice.slice_type}
      data-slice-variation={slice.variation}
    >
      <div className="grid gap-x-8 gap-y-6 md:grid-cols-[2fr,1fr]">
        <Heading size="xl" className="col-start-1">
          {slice.primary.heading}
        </Heading>

        <div className="prose prose-xl prose-slate prose-invert col-start-1">
          <PrismicRichText field={slice.primary.description} />
        </div>
    

     
      </div>
    </Bounded>
  );
};

export default Biography;

![image](https://github.com/a-trost/creative-portfolio/assets/104340985/80374191-1b38-4d3d-a2a7-e2bf9f5a916d)

![image](https://github.com/a-trost/creative-portfolio/assets/104340985/ac9bdceb-12c6-4189-93c1-225330260490)

here are two difference images for comparison , and i also created slices in remote web application as shown in video. Please help me if you found the error.

**Comment by RabinChakraborty** [comment]
hey, @HaribolGM I was suffering with the same issue before and was unable to solve it as well so instead I just followed what @a-trost did and used the key text field instead of the rich text field and the site is working now.

**Comment by HaribolGM** [comment]
> Hey, @HaribolGM I was suffering with the same issue before and was unable to solve it as well so instead I just followed what @a-trost did and used the key text field instead of the rich text field and the site is working now.

Yeah that's the problem i used normal text filed rather than rich text filed.
